Also, here is the required patches to solve some linking
 issue.
Patches by biswa96
---
 gtk2_ardour/wscript | 2 +-
 libs/ardour/wscript | 2 ++
 libs/fst/wscript    | 2 ++

diff --git a/gtk2_ardour/wscript b/gtk2_ardour/wscript
index 0869f99a23..c2dcdf5600 100644
--- a/gtk2_ardour/wscript
+++ b/gtk2_ardour/wscript
@@ -673,7 +673,7 @@ def build(bld):
         obj.source += [ 'lv2_plugin_ui.cc' ]
         obj.use += [ 'SUIL' ]
 
-    if bld.is_defined('HAVE_FFTW35F') and bld.env['build_target'] != 'mingw':
+    if bld.is_defined('HAVE_FFTW35F'):
         bld.env['LIB_FFTW3F'] += ['fftw3f_threads']
 
     if bld.is_defined('NEED_INTL'):
diff --git a/libs/ardour/wscript b/libs/ardour/wscript
index 57494c0c9e..7ccf50ebbe 100644
--- a/libs/ardour/wscript
+++ b/libs/ardour/wscript
@@ -355,6 +355,7 @@ def configure(conf):
 
     if conf.env['WINDOWS_VST_SUPPORT'] == True:
         conf.check(compiler='cxx', lib='gdi32', mandatory=True, uselib_store='GDI32')
+    conf.check(compiler='cxx', lib='bcrypt', mandatory=True, uselib_store='BCRYPT')
 
 def build(bld):
     # Library version (UNIX style major, minor, micro)
@@ -436,6 +437,7 @@ def build(bld):
         obj.includes += [ '../fst' ]
         obj.use += [ 'vstwin' ]
         obj.uselib += ['GDI32']
+    obj.uselib += ['BCRYPT']
 
     if bld.is_defined('LXVST_SUPPORT'):
         obj.source += [ 'lxvst_plugin.cc', 'linux_vst_support.cc' ]
diff --git a/libs/fst/wscript b/libs/fst/wscript
index 725cd9ec15..df174a38ef 100644
--- a/libs/fst/wscript
+++ b/libs/fst/wscript
@@ -12,6 +12,7 @@ def options(opt):
 def configure(conf):
     if conf.env['WINDOWS_VST_SUPPORT'] == True:
         conf.check(compiler='cxx', lib='gdi32', mandatory=True, uselib_store='GDI32')
+    conf.check(compiler='cxx', lib='bcrypt', mandatory=True, uselib_store='BCRYPT')
 
 def build(bld):
     if bld.is_defined('VST3_SUPPORT'):
@@ -35,6 +36,7 @@ def build(bld):
             obj.uselib += ' OSX'
         if bld.env['build_target'] == 'mingw':
             obj.uselib += ' GDI32'
+            obj.uselib += ' BCRYPT'
             obj.linkflags = ['-mwindows']
 
         if re.search ("bsd", sys.platform) != None:
- 
2.43.2

